src/main/java/org/apache/commons/io/file/Counters.java

```diff
@@ -92,8 +92,8 @@ public int hashCode() {
 
         @Override
         public String toString() {
-            return String.format("%,d files, %,d directories, %,d bytes", Long.valueOf(fileCounter.get()),
-                    Long.valueOf(directoryCounter.get()), Long.valueOf(byteCounter.get()));
+            return String.format("%,d files, %,d directories, %,d bytes", fileCounter.get(),
+                    directoryCounter.get(), byteCounter.get());
```

Comment:
Same comment as in your other PR. Let's leave boxing and unboxing as obvious since we are in a low level library and I prefer to see where this happens so we can think about consequences of creating objects.

Reply:
OK, will not happen next time :-)